iron-fit-behavior.html

```diff
@@ -253,11 +253,13 @@
       if (typeof this._isRTL === 'undefined') {
         this._isRTL = window.getComputedStyle(this).direction == 'rtl';
       }
-          
       this.positionTarget = this.positionTarget || this._defaultPositionTarget;
       if (this.autoFitOnAttach) {
-        if (window.getComputedStyle(this).display === 'none') {
-          setTimeout(function() {
+        // If invisible or in a shadow root that's still not connected, wait a tick.
+        if (window.getComputedStyle(this).display === 'none' ||
+          document.body.contains(this) === false) {
```

Comment:
updated to flush in https://github.com/PolymerElements/iron-fit-behavior/pull/64/commits/b6f544154225257b2c137fbfaa7385ce9989a2b8